### PR TITLE
Remove confusion in h1 to h6 implied mapping wording

### LINKED ISSUE #308: Potential for confusion in h1 to h6 implied mapping wording
The Implicit ARIA semantics column for [h1 to h6](https://w3c.github.io/html-aria/#el-h1-h6) currently reads:
> role=heading, with the aria-level = positive integer.

I have seen this misinterpreted as "role=heading, _if_ the aria-level = positive integer."

I think the following is less ambiguous (delete "with the"):
> role=heading, aria-level = positive integer.

If we use some of the words from HTML-AAM, the following may be even clearer?
> role=heading, aria-level = the number in the element's tag name.

I'll provide a PR with the 2nd wording, but I'm happy to change it to the shorter wording if that's preferred.

### PR DESCRIPTION
**Pull request by carmacleod**
- resolves #308

***
<a href="https://pr-preview.s3.amazonaws.com/carmacleod/html-aria/pull/309.html" title="Last updated on Apr 28, 2021, 7:28 AM UTC (86f04e2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/309/b65efa2...carmacleod:86f04e2.html" title="Last updated on Apr 28, 2021, 7:28 AM UTC (86f04e2)">Diff</a>

### CODE REVIEW

**Comment by scottaohara** [comment]
@carmacleod i like this change, thank you.

Unfortunately something was broken with the latest respec update so this spec keeps getting travis build failures for no logical reason.

**Review by patrickhlauke — APPROVED** [review]

**Comment by patrickhlauke** [comment]
respec issue fixed, so merging :+1:
